scrapy/utils/_compression.py

```diff
@@ -5,6 +5,19 @@
     import brotli
 except ImportError:
     pass
+else:
+    try:
+        brotli.Decompressor.process
+    except AttributeError:
+
+        def _brotli_decompress(decompressor, data):
```

Comment:
Should we add a warning here?

By the way `brotlipy` is now `brotlicffi` (with the import name adjusted accordingly), but whether we need to add support for it is a separate question.

Reply:
[Sounds like support may make sense for PyPy support](https://github.com/urllib3/urllib3/issues/2040), but indeed out of scope here.